src/main/scala/uk/gov/hmrc/ct/computations/CP298.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.computations
+
+import uk.gov.hmrc.ct.box.{Calculated, CtBoxIdentifier, CtOptionalInteger}
+import uk.gov.hmrc.ct.computations.calculations.SBACalculator
+import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
+
+case class CP298(value: Option[Int]) extends CtBoxIdentifier with CtOptionalInteger
```

Comment:
There was a change to CP296 which made it return all cost rather than the claim, so I'm not sure this is needed